src/headers.rs

```diff
@@ -112,8 +112,28 @@ impl TryFrom<&[u8]> for Headers {
         };
 
         if let Some(line) = lines.next() {
-            if !line.starts_with("NATS/") {
-                return parse_error("version line does not begin with NATS/");
+            let mut parts = line.splitn(3, ' ');
+
+            if let Some(v) = parts.next() {
+                if !v.starts_with("NATS") {
```

Comment:
Rust ecosystem more about inline or making those things const? I did make them const but not sure what is idiomatic these days.

Reply:
Constants was a good call 🙂